### PR TITLE
vmui: display partial response warning

### PR DESCRIPTION
**Pull request by Loori-R**
Added partial response warning

Screenshot 2023-07-30 at 15 07 49

#4721

### CODE REVIEW

**Review by dmitryk-dk — APPROVED** [review]
LGTM!

File: app/vmui/packages/vmui/src/components/Configurators/QueryEditor/warningText.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export const seriesFetchedWarning = `No match! 
+This query hasn't selected any time series from database.
+Either the requested metrics are missing in the database,
+or there is a typo in series selector.`;
+
+export const partialWarning = `The results you are viewing are marked as PARTIAL.
```
Comment (hagen1778) [inline app/vmui/packages/vmui/src/components/Configurators/QueryEditor/warningText.ts:6]:
```suggestion
export const partialWarning = `The shown results are marked as PARTIAL.
```

File: app/vmui/packages/vmui/src/components/Configurators/QueryEditor/warningText.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+This is due to one or more vmstorage nodes currently being unavailable.`;
```
Comment (hagen1778) [inline app/vmui/packages/vmui/src/components/Configurators/QueryEditor/warningText.ts:7]:
```suggestion
The result is marked as partial if one or more vmstorage nodes failed to respond to the query.`;
```

**Review by hagen1778 — COMMENTED** [review]
Looks good! Would you mind adding a note to the changelog?

**Review by hagen1778 — APPROVED** [review]
LGTM!

**Pull request merged**